gnomad/utils/file_utils.py

```diff
@@ -100,23 +102,62 @@ def parallel_file_exists(fpaths: List[str], parallelism: int = 750) -> Dict[str,
     )
 
 
-def file_exists(fname: str) -> bool:
+def file_exists(fname: str, overwrite: Optional[bool] = None) -> bool:
     """
     Check whether a file exists.
 
     Supports either local or Google cloud (gs://) paths.
-    If the file is a Hail file (.ht, .mt extensions), it checks that _SUCCESS is present.
+    If the file is a Hail file (.ht, .mt, .bm, .parquet, and .vds extensions), it checks that _SUCCESS is present.
 
-    :param fname: File name
-    :return: Whether the file exists
+    :param fname: File name.
+    :param overwrite: Optional boolean that will raise an error if set to False and file exists.
```

Comment:
```suggestion
    :param overwrite: Optional boolean that will raise an error if set to False and file doesn't exist.
```

Reply:
Actually in this case I want to raise the error if it does exist. Use case is when the file exists and we are trying to overwrite it, but we forgot to specify overwrite=True

Reply:
What if I just want to check if any file in the list doesn't exist and which are they? Setting `overwrite` to False to figure out make thing complicated. This question may be asked more, because the function doesn't overwrite anything.

Reply:
This function isn't for a list, it's just for a single file. By default the `overwrite` parameter is None and will not be used at all, this will then raise no error, it will simply do what it did before and return True or False. I will change to make another function that calls this one and has a clearer name that includes overwrite, I was just trying to put it into a single function. For the other function you again don't need to pass in overwrite.